clang/test/Interpreter/execute-pch.cpp

```diff
@@ -0,0 +1,14 @@
+// REQUIRES: host-supports-jit
+// UNSUPPORTED: system-aix
+
+// RUN: rm -f %t.pch
+// RUN: %clang_cc1 -fmax-type-align=16 -pic-level 2 -fdeprecated-macro -stack-protector 1 -fblocks -fskip-odr-check-in-gmf -fexceptions -fcxx-exceptions -fgnuc-version=0 -triple=%target_triple -DPCH -fincremental-extensions -emit-pch -x c++-header -o %t.pch %s
+// RUN: clang-repl -Xcc -fgnuc-version=0 -Xcc -triple=%target_triple -Xcc -include-pch -Xcc %t.pch '#include "%s"' | FileCheck %s
```

Comment:
@weliveindetail, it seems that clang-repl prefers the process triple:

https://github.com/llvm/llvm-project/blob/c26a99384bce5719107d26f4617d6e3b1e9253ff/clang/lib/Interpreter/Interpreter.cpp#L188

What should we do here? Should we implement a lit version of `host-supports-jit` but `host-jit-triple`? Is there something smarter we could do?

Reply:
When `-triple` is passed explicitly I'd expect it to override the process triple. Is there a reason not to do that?